src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -335,20 +335,20 @@ public void init() {
         logger.info("Entered REST addOwner...");
         return ResponseEntity
                 .ok()
-                .body(memberAttributeService.assignOwnership(path, currentUser, uid));
+                .body(membershipService.assignOwnership(path, currentUser, uid));
     }
 
     /**
      * Delete a grouping owner.
      */
-    @DeleteMapping(value = "/groupings/{path:[\\w-:.]+}/owners/{uid:[\\w-:.]+}")
-    public ResponseEntity<GroupingsServiceResult> deleteOwner(@RequestHeader("current_user") String currentUser,
+    @DeleteMapping(value = "/groupings/{path:[\\w-:.]+}/removeOwners/{uid}")
```

Comment:
The path can be changed to just /owners/ instead of /removeOwners/, because that is the RESTfull format and the HTML DELETE function already implies removal. You might want to make {uid} plural to indicate that multiple owners can be removed as well.